CONTRIBUTING.md

```diff
@@ -55,4 +58,130 @@ We prefer all communications to be in English.
 
 Microsoft follows the principle of [Coordinated Vulnerability Disclosure](https://www.microsoft.com/en-us/msrc/cvd).
 
-<!-- END MICROSOFT SECURITY.MD BLOCK -->
\ No newline at end of file
+<!-- END MICROSOFT SECURITY.MD BLOCK -->
+
+# Develop for CBL-Mariner
+When starting to develop for CBL-Mariner, use the [CBL-MarinerTutorials](https://github.com/microsoft/CBL-MarinerTutorials) repo. This repository guides developers on using CBL-Mariner's tools to customize or add new packages or images. Once you have confirmed your change builds and functions as expected, consider whether it should be added to the core repo, [CBL-Mariner](https://github.com/microsoft/CBL-Mariner).
+Please see either our [quickstart](toolkit/docs/quick_start/quickstart.md) or [building instructions](toolkit/docs/building/building.md) for technical instructions on building with CBL-Mariner.
+Please adhere to the [Pull Request guidelines](pull-request-guidelines) when contributing. 
+
+## Packages
+CBL-Mariner packages live in either [SPECS](SPECS) or [SPECS-EXTENDED](SPECS-EXTENDED). CBL-Mariner's core packages (`SPECS`) have full support and coverage. Packages in `SPECS-EXTENDED` are only guaranteed to compile successfully. `SPECS-EXTENDED` can be used as a staging area for iterating on packages with the possiblity of the package being graduated to `SPECS`. 
+
+| Package Support Level | Published | Supported | Comments 
+|:----------------------|:----------|:--------|:------------
+|SPECS-EXTENDED         |**Yes**    |No       | - Package needs a viable upstream source which actively addresses CVEs <br> - Package must not include project specific code
+|SPECS                  |**Yes**    |**Yes**  | - Package needs a viable upstream source which actively addresses CVEs <br> - Package must not include project specific code <br> - Package needs to offer value for multiple use cases
+
+When looking to graduate a package from `SPECS` to `SPECS-EXTENDED`, consult with a CBL-Mariner dev to highlight the package's value and ensure that the following steps are completed.
```

Comment:
you mean the opposite here? graduate from SPECS-EXTENDED to SPECS

Reply:
Yes, thank you!